checks.d/sqlserver.py

```diff
@@ -329,7 +329,6 @@ def open_db_connections(self, instance):
         except Exception as e:
             cx = "%s - %s" % (host, database)
             message = "Unable to connect to SQL Server for instance %s." % cx
-            self.log.warning("%s Exception: %s", message, e)
```

Comment:
Was it the exception that contained the connection string, and leaked the password? If not we could leave out the exception body, and just print out the `cx`? I think this integration is multi-instance....